### PR TITLE
Version padding for 15.8

### PR DESCRIPTION
**Pull request by tmat**
Keep this change in 15.8 branch only.

### CODE REVIEW

File: build/Versions.props [hunk 1]
```diff
@@ -4,6 +4,13 @@
     <VersionBase Condition="'$(UseVisualStudioVersion)' != 'true'">2.8.0</VersionBase>
     <PreReleaseVersionLabel>beta2</PreReleaseVersionLabel>
 
+    <!-- 
+      15.8 specific workaround for a change to BUILD_BUILDNUMBER format that added two more digits.
```
Comment (jasonmalinowski) [inline build/Versions.props]: Tagging @dpoeschl specifically since this will require work to not take it when merging back to master.

Comment (jasonmalinowski) [reply build/Versions.props]: (although I guess it's written in a way that will be fine still...)

Comment (tmat) [reply build/Versions.props]: Let's not merge it to master.

**Comment by davkean** [comment]
Can we front load the merge so its not accidentally merged? What about 15.9?

**Comment by dpoeschl** [comment]
@davkean It's included in https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=1852025 which I'm making an insertion for currently.

**Comment by dpoeschl** [comment]
Don't know about 15.9.

**Comment by dpoeschl** [comment]
Actually now that I've said all that, I don't know what you meant. "front load the merge so its not accidentally merged"?

**Comment by davkean** [comment]
What I mean is that we're going to get an auto-merge from 15.8 -> 15.9. Are we supposed to throw the commit away? I'd prefer we merge than revert.

**Comment by tmat** [comment]
The change should only be in 15.8. Not in 15.9 nor master.

**Comment by tmat** [comment]
merging and reverting the commit sounds good to me

**Comment by dpoeschl** [comment]
So let it flow to 15.9 and then revert it there before it gets inserted? (noting that 15.9 insertions haven't even been on my radar so we have time)?

**Comment by davkean** [comment]
I've gone ahead and done it here: https://github.com/dotnet/project-system/pull/3708 so that we don't forget about it.

**Pull request merged**